mandatory/2-logic-error.js

```diff
@@ -1,16 +1,18 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
-function getStringLength(word) {
-  return "word".length();
+var sentence = "A wild sentence appeared!";
+var word = "Turtles";
```

Comment:
Your variable is declared correctly. I would recommend using _let_ instead. Because if you work with someone else on a very big piece of code, another person might come up with the same variable name _word_ and without knowing rewrite yours. It will then can then cause a problem.